ntttcp/ntttcp.TESTGEN.ps1

```diff
@@ -130,8 +130,9 @@ function test_ntttcp {
 
     #Load the variables needed to generate the commands
     # execution time in seconds
-    [int] $g_runtime = 90
+    [int] $g_runtime = 60
     [int] $g_ptime   = 2
+    [int]    $g_bufferlen    = 1450
```

Comment:
Should we have different buffer lengths for udp and tcp? 1450 is ok for udp but seems too low for tcp.

Reply:
Yes, I will run another variance check to confirm that the variance is low for that buffer len.